recOrder/tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from recOrder.cli.settings import TransferFunctionSettings
+from iohub.ngff import open_ome_zarr
+
+@pytest.fixture(scope="function")
+def setup_default_ctf_settings(tmp_path):
+    settings = TransferFunctionSettings()
+    dataset = open_ome_zarr(tmp_path, layout="fov", mode="w", channel_names=["None"])
+    yield settings, dataset
+
+@pytest.fixture(scope="function")
+def setup_b_false_ctf_settings(tmp_path):
```

Comment:
Please use a more descriptive name for birefringence and phase instead of 'b' and 'p'.

Reply:
Also 'ctf' is not something obvious. If you think the function name is too long you can remove 'setup_'.

Since you are yielding a tuple of (settings, OTF) probably reorder it to reflect the sequence.

Reply:
I'm not quite sure what you mean by reorder it to reflect the sequence. Does that mean to make the name something like `settings_compute_transfer_function`?

Reply:
Yeah something like `settings_and_transfer_function`.